test/resources/Hooks/useCopied/index.test.js

```diff
@@ -0,0 +1,53 @@
+/** @jest-environment jsdom */
+import { renderHook, act } from '@testing-library/react'
+import useCopiedMessage from '../../../../resources/Hooks/useCopiedMessage'
+import link from '../../../../resources/link'
+
+jest.mock('../../../../resources/link', () => ({
+  send: jest.fn()
+}))
+
+describe('#useCopied', () => {
+  jest.useFakeTimers()
+
+  let result
+
+  const getBool = () => result.current[0]
+  const copyFn = () => result.current[1]()
+
+  beforeEach(() => {
+    ;({ result } = renderHook(() => useCopiedMessage('VALUE')))
```

Comment:
this callback function should be a test component we can interact with. so probably just a simple button that copies the message